Formula/git-annex.rb

```diff
@@ -28,15 +28,29 @@ class GitAnnex < Formula
   depends_on "quvi"
 
   def install
-    install_cabal_package :using => ["alex", "happy", "c2hs"], :flags => ["webapp"] do
-      # this can be made the default behavior again once git-union-merge builds properly when bottling
-      if build.with? "git-union-merge"
-        system "make", "git-union-merge", "PREFIX=#{prefix}"
-        bin.install "git-union-merge"
-        system "make", "git-union-merge.1", "PREFIX=#{prefix}"
+    # Outermost `cabal_sandbox do` can be removed when >aws-0.13.0 is released
+    cabal_sandbox do
+      # Remove when >aws-0.13.0 is released
+      # Gets S3 support working again; aws upstream has already patched this:
+      # https://github.com/aristidb/aws/commit/402bfe5aa9ef4bec84186880faafcbfdae1ad91d
+      # https://github.com/Homebrew/homebrew-core/issues/1268
+      # https://github.com/Homebrew/legacy-homebrew/issues/47737
+      system Formula["cabal-install"].opt_bin/"cabal", "get", "aws"
```

Comment:
@mikemcquaid I'm thinking we may want `cabal_get` in `language/haskell.rb` to avoid this "manual" invocation.